google-cloud-kms/acceptance/google/cloud/kms/v1/smoke_test.rb

```diff
@@ -20,7 +20,7 @@
 class KmsServiceSmokeTest < Minitest::Test
   def test_list_key_rings
     kms = Google::Cloud::Kms.key_management_service version: :v1
-    key_ring_parent = kms.class.location_path ENV["KMS_PROJECT"], "us-central1"
+    key_ring_parent = kms.location_path project: ENV["KMS_PROJECT"], location: "us-central1"
     key_rings = kms.list_key_rings(parent: key_ring_parent).to_a
     key_rings.size.must_equal 1
```

Comment:
I suspect the current failure is related to this usage being out of date. Change this to

```ruby
_(key_rings.size).must_equal 1
```

Similar for the following line.

Reply:
Okay but just changing this wasn't working and I was getting error `undefined method _` . Changing the parent class to `Minitest::Spec` fixed the error [ref](https://github.com/seattlerb/minitest/issues/651).

Reply:
Oh, my bad, I didn't notice it was trying to use expectations in a unit test. Sounds good, thanks!